data/lisbon.yaml

```diff
@@ -0,0 +1,35 @@
+---
+info:
+  city: Lisbon
+  link: lisbon
+  date: 12/17/2019
+  hour: '19:00'
+  icon: icons/lisbon/lisbon.svg
+  iconHover: icons/lisbon/lisbon-hover.svg
+  hostIcon: icons/paris/dashlane.svg
+  hostName: Dashlane
+  maxCapacity: 25
+site:
+  city: Paris
+  location: Dashlane
+  organizers:
+  googleMapsLink: https://goo.gl/maps/M8HCf3JqZ787HUzi8
+  calendarLink: https://www.google.com/calendar/render?action=TEMPLATE&text=QueerJS+Lisbon&location=Dashlane+R.+Garrett+12+2%C2%BA+piso%2C+1200-219+Lisboa%2C+Portugal&dates=20191217T180000Z%2F20191217T210000Z
+mainOrganizer:
+  - name: Paola Ducolin
+    main: true
+    phoneNumber: '+33 783059652'
+    twitterHandle: PolaDuco
+    email: paola.ducolin@dashlane.com
+thanks: []
+sponsors:
+  - name: Dashlane
+    link: https://www.dashlane.com/
+    media: https://cdn.dribbble.com/users/2023831/screenshots/4438878/dashlane-app-icon.png
+  - name: Heroku
+    link: https://www.heroku.com/
+    media: https://i.imgur.com/2BONYjO.png
+speakers:
+  - name: Paola Ducolin
```

Comment:
For QueerJS, we've made a point about wanting to feature new voices on stage with every meetup. While we're super excited to have you on stage, I would say it's better to choose one. So either speaking at Paris or Lisbon - but not both.

Reply:
Ok, I was not aware. Then I would keep Paris!

Reply:
Yeah no problem - we haven't really communicated it but had a conversation about it yesterday. I mean, if there ends up being a shortage of speakers in Lisbon then of course it'd be great if you could step in but let's maybe keep it as a backup 😊

Reply:
ok sure!

Reply:
A thought: You could MC??? No pressure, just an option 😊